android/res/layout/layout_nav_bottom_numbers.xml

```diff
@@ -17,8 +17,8 @@
       android:background="@drawable/speed_cams_bg"
       android:gravity="center"
       android:minWidth="@dimen/nav_numbers_side_min_width"
-      android:paddingStart="@dimen/nav_numbers_margin"
-      android:paddingEnd="@dimen/nav_numbers_margin">
+      android:paddingEnd="@dimen/nav_numbers_margin"
```

Comment:
These changes look unnecessary.

Reply:
I didn't make those changes. I was editing the file in the visual editor. For everything else you can blame Google and JetBrains. The formatting is applied using the project formatter for Android Studio you mentioned above.

Reply:
It doesn't matter who made these changes. Our repository doesn't need them. There are zero benefits in merging them. It pollutes the project git history. Please undo them.

We generally don't apply automatic code formatters, but stick to the defined code style.

Reply:
I have reformatted this file locally and it doesn't add extra spaces. So it could be that he has non-default configuration for xml formatting or Android Studio is doing that when adjusting files by visual editor.

Reply:
I understand and agree with your comments. But, unfortunately, I really don't have time at the moment to fiddle around with XML that was modified by the default tool for the job, using default or project settings as advised.

Reply:
I started looking into showing speed limit too, try reset the XML, then remove `android:paddingStart` & `android:paddingEnd`lines from the speed number element